docs/docs/guides/integrate/login/saml.md

```diff
@@ -0,0 +1,284 @@
+				<AttributeValue>Stark</AttributeValue>
+			</Attribute>
+			<Attribute Name="FirstName" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic">
+				<AttributeValue>Tony</AttributeValue>
+			</Attribute>
+			<Attribute Name="FullName" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic">
+				<AttributeValue>Tony Stark</AttributeValue>
+			</Attribute>
+			<Attribute Name="UserName" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic">
+				<AttributeValue>tony.stark@gmail.com</AttributeValue>
+			</Attribute>
+			<Attribute Name="UserID" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic">
+				<AttributeValue>260242264868201995</AttributeValue>
+			</Attribute>
+		</AttributeStatement>
+	</Assertion>
+</Response>
+```
+
+In this SAML response:
+
+- The **ID** (_164ba12b-6711-40e0-8ddb-55aa810f1c92) matches the ID from the SAML request (id-8LjuzBEUQFYFWjL55), allowing the service provider to verify it.
+- **InResponseTo** references the ID of the SAML request.
+- **IssueInstant** (2024-06-11T04:17:41Z) is the timestamp of the response.
+- **Destination** (http://127.0.0.1:5000/acs) confirms that the response is intended for the service provider's Assertion Consumer Service URL.
+- **Issuer** (https://my-instance-xtzfbc.zitadel.cloud/saml/v2/metadata) is the identity provider's unique string, used to verify the response's origin.
+- **StatusCode** indicates the status of the authentication process, with a value of Success.
+- **Signature** ensures the integrity and authenticity of the SAML response. It includes:
+- **SignedInfo** contains details about the canonicalization and signature methods.
+- **Reference** points to the signed data and including transforms and a digest method.
+- **SignatureValue** provides the actual digital signature.
+- **KeyInfo** contains the X.509 certificate, which holds the public key used to verify the signature.
+- **Assertion** includes user information, such as email, surname, first name, full name, username, user ID etc.
+- **Subject** contains the user's unique identifier.
+- **Conditions** specify the response's validity window.
+- **AuthnStatement** includes authentication details.
+- **AttributeStatement** contains additional user attributes.
+
+## SAML identity brokering
+
+### How SAML identity brokering works
+
+- **Initial Authentication Request**: A user attempts to access a service (SP1) that is protected by an IdP (IdP1).
+- **Redirection to IdP1**: The user is redirected to IdP1 for authentication. If IdP1 trusts another IdP (IdP2) for authentication, it will redirect the user to IdP2.
+- **Authentication at IdP2**: The user authenticates with IdP2, which generates a SAML assertion containing the user's identity and attributes.
+- **Assertion Transfer**: IdP2 sends the SAML assertion back to IdP1. IdP1 processes this assertion and may add additional attributes or modify the assertion based on its own policies.
```

Comment:
There should be no transfer? If ZITADEL is IdP1, the Assertion from IdP2 is processed and an independent assertion is created and used as response to SP1.

Reply:
```suggestion
- **Assertion Processing**: ZITADEL (IdP1) processes the SAML assertion from IdP2. ZITADEL then creates an independent SAML assertion based on the information received and its own policies.
- **Response to SP1**: ZITADEL (IdP1) sends this newly created SAML assertion to SP1, completing the authentication process.
```